ecs_windows/README.md

```diff
@@ -0,0 +1,181 @@
+## Using default Amazon ECS configurations for Windows images
+
+In order to allow Amazon ECS Windows customers to get started promptly with Fluent Bit without performing significant configuration, we have crafted default configurations for the following output destinations-
+* [Amazon CloudWatch](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows/cloudwatch.conf)
+* [Amazon S3](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows/s3.conf)
+* [Amazon Kinesis Data Streams](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows/kinesis.conf)
+* [Amazon Kinesis Data Firehose](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows/firehose.conf)
+
+These default configurations would be baked in the Windows container images. In order to use the same out of the box with Amazon ECS, you need to ensure-
```

Comment:
"would be" => are

Reply:
We might want to put the version number of the first release with these

Reply:
Did this change.